lib/peek/controller_helpers.rb

```diff
@@ -9,6 +9,11 @@ module ControllerHelpers
 
     protected
 
+    def append_info_to_payload(payload)
+      super
+      payload[:peek_enabled] = request.cookies['peek'] == 'true'
```

Comment:
We should move this conditional to `peek_enabled?` below and let internal method determine what the boolean is in the payload.